crypto/digestible/derive/src/lib.rs

```diff
@@ -48,16 +158,14 @@ fn try_digestible_struct(
                 // this is a regular struct
                 Some(field_ident) => {
                     quote! {
-                        hasher.update(stringify!(#field_ident).as_bytes());
-                        self.#field_ident.digest(hasher);
+                        self.#field_ident.append_to_transcript_allow_omit(stringify!(#field_ident).as_bytes(), transcript);
                     }
                 }
                 // this is a tuple struct, and the member doesn't have an identifier
```

Comment:
This comment seems to use "member" and "field" interchangeably. Might be clearer as:
"This field is a tuple struct, and the field doesn't have an identifier.", or "This field is a tuple struct without an identifier."